modules/ROOT/pages/x12-send-settings.adoc

```diff
@@ -4,14 +4,15 @@ Partner Manager enables you to configure the X12 envelope parameters, delimiters
 
 == Configure the Send Settings
 
-. In the left navigation menu, select the partner for whom to configure the send settings.
-. In the *Validation and acknowledgment settings* section, under *Send to <partner-name>*, click *X12*.
-. In the menu on the left, select the setting to configure:
-* <<general-settings,*General*>>
-* <<authorization-settings,*Authorization*>>
-* <<ta1-ack-settings,*TA1 Acknowledgment*>>
+. In the navigation menu, select *Partners*.
+. Select the partner for whom you want to configure the settings.
+. In the *Validation and acknowledgment settings* section's *Send to <partner-name>* column, click *X12*.
+. In menu on the left, select the setting to configure:
```

Comment:
```suggestion
. In the menu on the left, select the setting to configure:
```

Reply:
Not sure if there is a way for you to remove "left."

Reply:
This is one I'm not sure what to do with yet. I am tracking this in a spreadsheet so I can go back to it.